raw_description/urdf/torso/torso_v2.urdf.xacro

```diff
@@ -14,29 +14,29 @@
 
     <link name="${name}_0_link">
       <inertial>
-        <origin xyz="0 0 0" rpy="0 0 0"/>
+        <origin xyz="0 0 0.254" rpy="0 0 0"/>
         <mass value="0.57307"/>
         <inertia ixx="1.415" ixy="0" ixz="0" iyy="0.819" iyz="0" izz="0.819" />
       </inertial>
```

Comment:
use [`cylinder_inertial`-macro](https://github.com/ipa320/cob_common/blob/indigo_dev/cob_description/urdf/common.xacro#L36-L44) instead

Reply:
If you can give me an example how to use that, yes. Otherwise we can stay with what already works.

Reply:
those are used as any other xacro:macros!
e.g. https://github.com/ipa320/cob_common/blob/f94b94803774ce219a6ed1cbfdd55c289ca9290d/cob_description/urdf/sensors/sick_s300_laser.urdf.xacro#L17